src/App.tsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination/Pagination';
+import './App.css';
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const total = 42;
+  const firstLiItem = perPage * (currentPage - 1) + 1;
+  const lastLiItem = perPage * currentPage < total
+    ? perPage * currentPage
+    : total;
+  const itemsPerPage = getNumbers(firstLiItem, lastLiItem);
```

Comment:
You have removed array of items from line 7, try to replace the hardcoded `42` with the length of the items array. For example, `const total = items.length`